lib/publiccloud/k8sbasetest.pm

```diff
@@ -51,7 +51,7 @@ sub get_k8s_service_name {
         return "GKE";
     }
     elsif ($provider eq 'AZURE') {
-        return "AKS";
```

Comment:
While I don't know the context here, it seems to me that AKS wasn't a typo

Reply:
Hello @olblak, true. I'm still working on this. There will be lot of such cases.